app/components/RoomList/index.js

```diff
@@ -339,6 +346,20 @@ class RoomList extends Component {
 									onClick={() => this.setState({ showRoomDeleter: true })}
 								/>
 							</Button.Group>
+							<span style={{ marginLeft: 16 }}>Status</span>
+							<Select
+								mode="multiple"
+								style={{ marginLeft: 16, width: '30%' }}
+								placeholder="Filter By Status..."
+								defaultValue={[ 'G','Y','R' ]}
+								onChange={e => this.setState({ statusFilter: e})}
```

Comment:
leave the `defaultValue` blank and just have the `value` depend on `statusFilter` from the state